package.json

```diff
@@ -53,12 +54,15 @@
 		"postcss-load-config": "^1.2.0",
 		"postcss-loader": "^2.0.5",
 		"resolve-url-loader": "^2.0.2",
-		"sass-lint": "^1.9.1",
+		"sass-lint": "^1.11.1",
 		"sass-loader": "^6.0.5",
 		"script-loader": "^0.7.0",
 		"url-loader": "^0.5.8",
 		"webpack": "^2.6.1"
 	},
+	"resolutions": {
+		"eslint": "^4.6.1"
```

Comment:
this forces eslint to update, because `sass-lint` is stuck on version 2.x and I do not see any intention to publish an update from them